EjercicioAlumno/src/FiguraTest.java

```diff
@@ -0,0 +1,49 @@
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class FiguraTest {
+
+    @Test
+    @DisplayName("Testeando area del cuadrado")
```

Comment:
Un nombre mas descriptivo sobre lo que hace el test.

"when calcularArea is called THEN it should set the correct area"

o

"when calcularArea is called with 4.0 THEN it should set the correct an area of 16"